src/Microsoft.DotNet.Interactive.HttpRequestParser/HttpEmbeddedExpressionNode.cs

```diff
@@ -17,13 +17,13 @@ internal class HttpEmbeddedExpressionNode : HttpSyntaxNode
         HttpExpressionEndNode endNode) : base(sourceText, syntaxTree)
     {
         StartNode = startNode;
-        Add(StartNode);
+        AddInternal(StartNode);
```

Comment:
Why does this not follow the same pattern with specific Add() overloads and private setters for the properties (as opposed to setting the properties in the constructor)?

Reply:
This was a rename refactor. I haven't carried that pattern through everywhere yet.

Reply:
Ah unless I missed something I think you have fixed up all the other node types to follow the pattern. This one seemed like the only one that wasn't fixed...

Reply:
Fixed in the next PR.